utils/build/docker/python/django-py3.13.base.Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN python --version && curl --version
 # install python deps
 ENV PIP_ROOT_USER_ACTION=ignore
 RUN pip install --upgrade pip
-RUN pip install django==5.2.6 pycryptodome==3.23.0 gunicorn==23.0.0 gevent==25.8.2 requests==2.32.4 boto3==1.34.141 'moto[ec2,s3,all]'==5.0.14 xmltodict==0.14.2 zope.event==5.1.1 zope.interface==7.2
+RUN pip install django==5.2.7 pycryptodome==3.23.0 gunicorn==23.0.0 gevent==25.9.1 requests==2.32.4 boto3==1.40.64 'moto[ec2,s3,all]'==5.1.16 xmltodict==0.14.2 zope.event==6.0 zope.interface==8.0.1
```

Comment:
can we leave a comment on why these specific versions matter? otherwise someone might come in here and bump them not knowing the impact

Reply:
I just updated to latest.